src/components/SignUpForm.jsx

```diff
@@ -50,9 +50,13 @@ const CreateAccount = (props, routeProps) => {
 				const NameFirst = response.data.Results.NameFirst;
 				const NameLast = response.data.Results.NameLast
 
-				props.setFieldValue('ContactID', ContactID);
-				props.setFieldValue('NameFirst', NameFirst);
-				props.setFieldValue('NameLast', NameLast);
+				const fields = {
```

Comment:
same comment as above this can be simplified.

Reply:
fixed